examples/templates/index.html

```diff
@@ -110,27 +114,27 @@ <h4>
 
 {% block extrascripts %}
   <script type="text/javascript">
-    $(function () {
-      // Log in & Sign up buttons
-      // The formURL is given explicitly
-      $(".simple-btn").modalForm({formURL: "{% url 'simple' %}"});
-      $(".login-btn").modalForm({formURL: "{% url 'login' %}"});
-      $(".signup-btn").modalForm({formURL: "{% url 'signup' %}"});
+      $(function () {
+          // Log in & Sign up buttons
+          // The formURL is given explicitly
+          $(".simple-btn").modalForm({formURL: "{% url 'simple' %}"});
```

Comment:
Simple form should be removed. This breaks index page, since the simple form url was removed. The simple.html is still in templates and on index page. I guess the instantiation of filter form is now also missing in index page js script.